frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -63,9 +51,9 @@ export function OngoingMissionCard({ mission }: MissionProps) {
         navigate(path)
     }
     return (
-        <StyledMissionCard variant="default" style={{ boxShadow: tokens.elevation.raised }}>
+        <StyledMissionCard style={{ boxShadow: tokens.elevation.raised }}>
             <TopContent>
-                <StyledTitle variant="default" onClick={routeChange}>
+                <StyledTitle onClick={routeChange}>
                     <StyledPadding>
```

Comment:
Would remove <StyledPadding>. Became a bit much in combination with the padding on StyledTitle